crmsh/utils.py

```diff
@@ -2649,4 +2649,46 @@ def ping_node(node):
     rc, _, err = get_stdout_stderr("ping -c 1 {}".format(node))
     if rc != 0:
         raise ValueError("host \"{}\" is unreachable: {}".format(node, err))
+
+
+def is_quorate(expected_votes, actual_votes):
+    """
+    Given expected votes and actual votes, calculate if is quorated
+    """
+    return int(actual_votes)/int(expected_votes) > 0.5
+
+
+def get_stdout_or_raise_error(cmd):
```

Comment:
I think we already have many methods like `get_stdout...`. Did you consider changing any of them?
In any case, this new method works fine too

Reply:
By the way, I would pass the `0` constant in the parameter being `0` default. Otherwise, you might have some command in the future that returns something different than `0` as good, and this method might not work properly

Reply:
> I think we already have many methods like `get_stdout...`. Did you consider changing any of them?
> In any case, this new method works fine too

@arbulu89 I don't know what's your point here? Do you mean I should give this function a new name not starts with `get_stdout`?

Reply:
No. I'm just saying that we have already multiple methods running comands and getting stdout, stderr, etc, and now we are adding a new one. I was just thinking loud, that we could have a generic method that handles all of them configuring with the incoming parameters. 
But forget about this by now hehe